pkg/varlinkapi/containers.go

```diff
@@ -100,24 +102,33 @@ func (i *LibpodAPI) ListContainerProcesses(call ioprojectatomicpodman.VarlinkCal
 	if err != nil {
 		return call.ReplyContainerNotFound(name)
 	}
-	containerState, err := ctr.State()
-	if err != nil {
-		return call.ReplyErrorOccurred(err.Error())
-	}
-	if containerState != libpod.ContainerStateRunning {
-		return call.ReplyErrorOccurred(fmt.Sprintf("container %s is not running", name))
+	if call.WantsMore() {
+		call.Continues = true
 	}
-	var psArgs []string
-	psOpts := []string{"-o", "uid,pid,ppid,c,stime,tname,time,cmd"}
-	if len(opts) > 1 {
-		psOpts = opts
-	}
-	psArgs = append(psArgs, psOpts...)
-	psOutput, err := ctr.GetContainerPidInformation(psArgs)
-	if err != nil {
-		return call.ReplyErrorOccurred(err.Error())
+	for {
+		containerState, err := ctr.State()
+		if err != nil {
+			return call.ReplyErrorOccurred(err.Error())
+		}
+		if containerState != libpod.ContainerStateRunning {
+			return call.ReplyErrorOccurred(fmt.Sprintf("container %s is not running", name))
+		}
+		var psArgs []string
+		psOpts := []string{"-o", "uid,pid,ppid,c,stime,tname,time,cmd"}
+		if len(opts) > 1 {
+			psOpts = opts
+		}
+		psArgs = append(psArgs, psOpts...)
```

Comment:
Do we need this? psArgs is always empty at this point, so you can probably just remove it and pass psOpts straight into GetContainerPidInformation

Reply:
sure, fixed